templates/cisco_wlc_ssh_show_stats_port_summary.textfsm

```diff
@@ -0,0 +1,23 @@
+Value PORT (\d)
```

Comment:
I would use this template. It is similar with that we did on one of the previous ones, but uses `Fillup`.

```textfsm
Value Required PORT (\d)
Value LINK_STATUS (\S+)
Value PACKETS_IN (\d+)
Value PACKETS_OUT (\d+)
Value BROADCAST_IN (\d+)
Value ERRORS_IN (\d+)
Value ERRORS_OUT (\d+)
Value COLLISIONS (\d+)
Value Fillup TOTAL_PACKETS_IN (\d+)
Value Fillup TOTAL_PACKETS_OUT (\d+)
Value Fillup TOTAL_BROADCAST_IN (\d+)
Value Fillup TOTAL_ERRORS_IN (\d+)
Value Fillup TOTAL_ERRORS_OUT (\d+)
Value Fillup TOTAL_COLLISIONS (\d+)

Start
  ^\s+Link\s+(Pkts In\s+){2}\s*Pkts Out\s*$$
  ^\s*Pr\s+Status\s+(Pkts (In|Out)\s+){2}\s*Bcast\s+(?:Errors\s+){2}\s*Collisions\s*$$ -> Port_Stats
  ^\s*$$
  ^. -> Error

Port_Stats
  ^-+
  ^${PORT}\s+${LINK_STATUS}\s+${PACKETS_IN}\s+${PACKETS_OUT}\s+${BROADCAST_IN}\s+${ERRORS_IN}\s+${ERRORS_OUT}\s+${COLLISIONS}\s*$$ -> Record
  ^\s+Total\s+${TOTAL_PACKETS_IN}\s+${TOTAL_PACKETS_OUT}\s+${TOTAL_BROADCAST_IN}\s+${TOTAL_ERRORS_IN}\s+${TOTAL_ERRORS_OUT}\s+${TOTAL_COLLISIONS}\s*$$
  ^\s*$$
  ^. -> Error
```

Reply:
@FragmentedPacket 
Sounds good.
I like the changes and appreciate the suggestion.
My branch has those changes made now. Thank you!